### DISCUSSION TITLE
Where is `wasiImport` in node `@wasmer/wasi` package?

### DISCUSSION THREAD

**Discussion by xcodedave** [wasmerio/wasmer#4799] [General]
We're building a `Vite` based node app, and trying to setup a basic `WebAssembly` instance which depends on `wasi_snapshot_preview1`.

Most of the docs and examples show that the `WASI` object should have a `wasiImport` prop that can be used to satisfy this dependency when instantiating via `WebAssembly.instantiateStreaming`.

e.g.:
with the following `@wasmer/wasi` package import:

```typescript
import { init, WASI } from '@wasmer/wasi'
```

the following code:
```typescript
await init()
const wasi = new WASI({
  env: {},
  args: [],
});
const moduleBytes = fetch(wasmUrl)
const imports = {
  wasi_snapshot_preview1: wasi.wasiImport
}
const instance = await WebAssembly.instantiateStreaming(moduleBytes, imports)
```

gives us this error:
```
Property 'wasiImport' does not exist on type 'WASI'.
```

Our `package-lock.json` shows that we are using the following `@wasmer/wasi` version:
```javascript
 "node_modules/@wasmer/wasi": {
      "version": "1.2.2",
      "resolved": "https://registry.npmjs.org/@wasmer/wasi/-/wasi-1.2.2.tgz",
      "integrity": "sha512-39ZB3gefOVhBmkhf7Ta79RRSV/emIV8LhdvcWhP/MOZEjMmtzoZWMzt7phdKj8CUXOze+AwbvGK60lKaKldn1w=="
 }
```

How are we supposed to provide the `wasi_snapshot_preview1` dependencies please?

**Comment by theduke** [comment]
cc @syrusakbary
